framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/OpenShiftDeployment.java

```diff
@@ -109,12 +110,12 @@ public boolean isReady() {
             String deploymentConfigName = getServiceName();
 
             List<Instance> instances = openShift.getPods().stream()
-                    .filter(pod -> {
-                        String podsDeploymentConfigName = pod.getMetadata().getLabels().get(OpenShiftResourceConstants.DEPLOYMENT_CONFIG_LABEL);
-                        return deploymentConfigName.equals(podsDeploymentConfigName);
-                    })
-                    .map(pod -> createInstance(pod))
-                    .collect(toList());
+                                                .filter(pod -> {
```

Comment:
Are these formatting changes coming from the kie-group IDE formatter?

Reply:
This was done by the formatter found here: https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/master/ide-configuration/eclipse-configuration/code-style

Reply:
That should be correct. One can argue that the previous formatting was better :)